javascript/javascript3/week1/GiphyApi/main.js

```diff
@@ -0,0 +1,53 @@
+const apiKey = 'TnGhS9Vu9jrTmAOpBbNXscaRtP8opJNw'
+const giphyName = document.getElementById('giphy-name');
+const btnsearch = document.getElementById('btn-giphy');
```

Comment:
please remember to use camelCase notation for  variable naming ... also try using descriptive names e.g 'gifCount' .

Reply:
Yes, your are right. I will be careful. I am having lots problem in descriptive names.